tests/garage/tf/core/test_networks.py

```diff
@@ -156,3 +156,26 @@ def test_layer_normalization(self):
             self.mlp_f_w_n, feed_dict={self._input: self.obs_input})
 
         np.testing.assert_array_almost_equal(out, mlp_output)
+
+    def test_param():
+        input_vars = tf.placeholder(shape=[None, 2, 3, 4], dtype=tf.float32)
+        initial_params = np.array([48, 21, 33])
+
+        init = tf.constant_initializer(initial_params)
+
+        params = parameter(
+            input_var=input_vars,
+            length=3,
+            initializer=init,
+        )
+
+        data = np.random.uniform(size=[5, 2, 3, 4])
```

Comment:
generally avoid using random values in tests unless you're actually testing randomness. it can lead to flaky tests.

Reply:
done